src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
+
 ?>
+<span class="tribe-events-c-small-cta-hellip"> &hellip; </span>
 <div class="tribe-events-c-small-cta tribe-common-b3 tribe-events-c-small-cta--readmore">
```

Comment:
2 notes here:
1. Do we have styles for `tribe-events-c-small-cta--readmore`? If not, we probably shouldn't have that class.
2. Since this is a read more component, it should have a read more component class (like `tribe-events-c-read-more` or something similar).

Reply:
Yes we have https://github.com/moderntribe/the-events-calendar/blob/release/B19.14/src/resources/postcss/components/skeleton/_small-cta.pcss#L20

Reply:
2. I treated this as a cta component (has the same styles) but with a modifier given that requires different spacing as a variation of the cta (it's a read more).

Reply:
So I never got a chance to review https://github.com/moderntribe/the-events-calendar/pull/2741, but if we're calling this template a read more component, we should be naming it as such. Not all of our CSS components have a corresponding template, just a template structure we use. Some are just styles we reuse throughout the code, much like `tribe-common-h1`, for example. So if we're reusing the `--read-more` modifier for the small CTA in other places, I'd agree. But since it's specific to the read more component, I'd be more inclined to put the style on the read more component class.

Reply:
Well, I see it differently. Since we're using this template only to override the twentyseventeen read more button and make it look like the CTA but with the difference of the margin, I though it would be pointless to define a new component, and it would be way more efficient to define a modifier, as it is a really really special case.